adagucserverEC/CServerConfig_CPPXSD.h

```diff
@@ -715,6 +717,27 @@ class CServerConfig : public CXMLSerializerInterface {
       } else if (equals("bordercolor", 11, attrname)) {
         attr.bordercolor.copy(attrvalue);
         return;
+      } else if (equals("labelfontfile", 13, attrname)) {
+        attr.labelfontfile.copy(attrvalue);
+        return;
+      } else if (equals("labelfontsize", 13, attrname)) {
+        attr.labelfontsize.copy(attrvalue);
```

Comment:
Can you make sure that these are described in https://github.com/KNMI/adaguc-server/blob/add_polygon_labels/doc/configuration/FeatureInterval.md ?

Reply:
Done